playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -624,6 +624,8 @@ EDXAPP_NEWRELIC_WORKERS_APPNAME: "{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT
 EDXAPP_ORA2_FILE_PREFIX: None
 EDXAPP_FILE_UPLOAD_STORAGE_BUCKET_NAME: '{{ EDXAPP_AWS_STORAGE_BUCKET_NAME }}'
 EDXAPP_FILE_UPLOAD_STORAGE_PREFIX: 'submissions_attachments'
+EDXAPP_ORA2_FILEUPLOAD_BACKEND: None
+EDXAPP_ORA2_FILEUPLOAD_CACHE_NAME: None
```

Comment:
Do we want `None` or the values you added in platform? like `'ora2-storage'`.

Reply:
Also, what happens in devstack vs prod/stage?

Reply:
In fact, do we even need the memcache stuff?

Reply:
Wait yea, actually I think we can just have it use the default django file storage...-.-